amplify/cli.json

```diff
@@ -1,3 +1,19 @@
 {
```

Comment:
Intentionally made executable? Wouldn't have expected this to be intentional for a json file!

Reply:
Seems wrong to me too, but it's generated by the Amplify CLI so I'm inclined to leave it as it is.